src/include/add-on/add-on-workflow.rb

```diff
@@ -224,37 +217,31 @@ def MediaSelect
         sources_after = Pkg.SourceGetCurrent(false)
         Builtins.y2milestone("Sources with new one added: %1", sources_after)
 
-        # bnc #393011
-        # AddOnProduct::src_id must be set to the latest source ID
-        src_id_found = false
+        @added_repos = []
```

Comment:
this method is already long as a week before payment. Is possible to move this `:next` handling to own method?